docs/user-event/intro.mdx

```diff
@@ -60,33 +60,37 @@ itself - e.g. in a `before`/`after` hook - for reasons described in
 
 ```js
 import userEvent from '@testing-library/user-event'
-import { render, screen } from '@testing-library/vue'
 
 // inlining
 test('trigger some awesome feature when clicking the button', async () => {
   const user = userEvent.setup()
+  // Import `render` from the framework library of your choice.
+  // See https://testing-library.com/docs/dom-testing-library/install#wrappers
   render(<MyComponent />)
 
-  await user.click(screen.getByRole('button', {name: /click me!/i}))
+  await user.click(screen.getByRole('button', { name: /click me!/i }))
 
   // ...assertions...
 })
 ```
 
 ```js
 import userEvent from '@testing-library/user-event'
-import { render } from '@testing-library/vue'
 
 // setup function
 function setup(jsx) {
   return {
     user: userEvent.setup(),
+    // Import `render` from the framework library of your choice.
+    // See https://testing-library.com/docs/dom-testing-library/install#wrappers
     ...render(jsx),
   }
 }
 
 test('render with a setup function', async () => {
-  const {user} = setup(<MyComponent />)
+  // Import `setup` from the framework library of your choice.
+  // See https://testing-library.com/docs/dom-testing-library/install#wrappers
```

Comment:
`setup` is declared above in the same snippet.

Reply:
Oops. This is what happens when you make these edits in a rush. Fixed.